### PR TITLE
Fix issues with sync-files script on mac and zsh

### PR DESCRIPTION
**Pull request by indytechcook**
I have errors when trying to sync files locally, this fixes theme

---
This change is [Reviewable](https://reviewable.io/reviews/department-of-veterans-affairs/va.gov-cms/1251)

### CODE REVIEW

File: scripts/sync-files.sh [hunk 1]
```diff
@@ -7,7 +7,8 @@ set -e
 cd $(git rev-parse --show-toplevel)/.dumps
 
 curl --remote-name --remote-header-name https://dsva-vagov-prod-cms-backup-sanitized.s3-us-gov-west-1.amazonaws.com/files/cms-prod-files-latest.tgz
-rm --force --recursive ../docroot/sites/default/files/*
+rm -rf ../docroot/sites/default/files/*
```
Comment (ElijahLynn) [inline scripts/sync-files.sh]: Ahh yes, BSD `rm` doesn't have the long options.

File: scripts/sync-files.sh [hunk 2]
```diff
@@ -7,7 +7,8 @@ set -e
...
 echo "Extracting files to sites/default/files."
+mkdir -p ../docroot/sites/default/files
```
Comment (ElijahLynn) [inline scripts/sync-files.sh]:
```suggestion
mkdir --parents ../docroot/sites/default/files
```

Comment (indytechcook) [reply scripts/sync-files.sh]:
```
~/code/va/va.gov-cms/.dumps VAGOV-1237-staged-revision*
❯ mkdir --parents ../docroot/sites/default/files
mkdir: illegal option -- -
usage: mkdir [-pv] [-m mode] directory ...
```

File: scripts/sync-files.sh (hunk 1 above)
Comment (oksana-c) [reply scripts/sync-files.sh]: @indytechcook , beat me to it 👍

**Pull request merged**